pyiron/vasp/base.py

```diff
@@ -1341,6 +1341,15 @@ def get_nelect(self):
         else:
             return self["output/generic/dft/n_elect"]
 
+    def get_magnetic_moments(self):
+        """
+        Gives the magnetic moments of a calculation for each iteration step.
+
+        Returns:
+            numpy.ndarray: array of final magmetic moments
+        """
+        return self['output/generic/dft/final_magmoms']
```

Comment:
To me it makes more sense that it behaves in the same way as `get_structure`, i.e. it has `snapshot` inside the argument which is set to -1.

Reply:
Fair enough! I also thought about it before.